src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV, I've already said above, that 'isReady' means that state **is ready** for complete work and that 'loaded' is not suitable, because we loading/fetching many things many times and it gets confused with isFetching flag. Each component will have its own implementation for initialization of this flag, so anyone will see what it exactly means for each component separately inside of it _componentDidMount_. We need common name to prevent fallible implementation of such logic. If you have better solution - please offer it.